### PR TITLE
update changelog

### LINKED ISSUE #246: Update changelog
I think changelog entries are missing for #243 and #244

https://standard.open-contracting.org/staging/profiles/ppp/1.0-dev/en/reference/changelog/

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes https://github.com/open-contracting-extensions/public-private-partnerships/issues/246

### CODE REVIEW

**Comment by duncandewhurst** [comment]
Failures are unrelated to this PR and are resolved in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245

**Review by jpmckinney — APPROVED** [review]

**Comment by duncandewhurst** [comment]
@jpmckinney can you merge? or do we need to wait until #245 is merged, then merge 1.0-dev into this branch?

**Comment by jpmckinney** [comment]
Sure, though 1.0-dev will start failing, because the new tests are being applied, which only #245 fixes.

**Pull request merged**
